modules/cbcsrf/models/Util.cfc

```diff
@@ -28,7 +28,9 @@ component accessors=true {
 
 		// Does this engine support CSRF?
 		if ( ( engine eq CFMLEngine.ADOBE && version gte 10 )
-			 || ( engine eq CFMLEngine.RAILO && version gte 4 ) ){
+			 || ( engine eq CFMLEngine.LUCEE && version gte 4 )
```

Comment:
All engines support this now since CF10. The problem is that some of them do not support clustering setups. So I would say to default the isCSRF to false and use the implementation that Brad built.  If somebody wants to use the native methods, they can do so easily on their own.

Reply:
What implementation did Brad build?

Reply:
@gpickin This repo is what I built :)